### PR TITLE
Added ability to treat Postgres NUMERIC as double

### PR DESCRIPTION
**Pull request by ssiahetiong**

### CODE REVIEW

**Comment by jpechane** [comment]
Looks good to me, but from understandabilty POV it might be better to split it into two commits - the chnage it self and formatting changes - @gunnarmorling, @rhauch WDYT?

**Comment by ssiahetiong** [comment]
Amended commit to discard formatting changes

**Comment by gunnarmorling** [comment]
Hey @ssiahetiong, finally getting to look into this. Sorry for the delay. It looks good overall, thanks a lot.

Could you only file a JIRA issue in [our tracker](https://issues.jboss.org/browse/DBZ) and reword the commit message so it begins with the issue key ("DBZ-xyz Adding ability ..."). And could you also use the e-mail address of your GitHub account for the commit? That way it is [properly linked](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user) to your user in the history.

Thanks!

File: debezium-connector-postgres/src/test/java/io/debezium/connector/postgresql/RecordsStreamProducerIT.java [hunk 1]
```diff
@@ -279,6 +281,26 @@ public void shouldReceiveChangesForDeletes() throws Exception {
...
+        TestHelper.executeDDL("postgres_create_tables.ddl");
+
+        consumer = testConsumer(1);
+        recordsProducer.start(consumer);
+
+        List<SchemaAndValueField> schemasAndValuesForNumericType = schemasAndValuesForNumericType();
+        schemasAndValuesForNumericType.set(3, new SchemaAndValueField("d", Schema.OPTIONAL_FLOAT64_SCHEMA, 1.1d));
+        schemasAndValuesForNumericType.set(4, new SchemaAndValueField("n", Schema.OPTIONAL_FLOAT64_SCHEMA, 22.22d));
+
+        assertInsert(INSERT_NUMERIC_TYPES_STMT, schemasAndValuesForNumericType);
```
Comment (gunnarmorling) [inline debezium-connector-postgres/src/test/java/io/debezium/connector/postgresql/RecordsStreamProducerIT.java]: @ssiahetiong, I'm getting an exception due to the mismatch of the `BigDecimal` value being passed which isn't expected by Connect for the FLOAT64 column.

I'm surprised that the proposed change doesn't contain a change to `PostgresValueConverter.convertDecimal()` which I think should emit a double to make happen what you intend.

Perhaps you can share some more insights on your testing?

Comment (ssiahetiong) [inline debezium-connector-postgres/src/test/java/io/debezium/connector/postgresql/RecordsStreamProducerIT.java]: When I made these changes, there was still no convertDecimal method. The new methid might have affected the unit test. I need to rebase this to master to check it out .

Comment (gunnarmorling) [inline debezium-connector-postgres/src/test/java/io/debezium/connector/postgresql/RecordsStreamProducerIT.java]: Got it. I've rebased your change and adapted it slightly, it's `PostgresValueConverter#converter()` now which must return the correct conversion method based on the decimal handling mode. With that, the test passes.

**Comment by gunnarmorling** [comment]
Rebased and applied. Also filed [DBZ-337](https://issues.jboss.org/browse/DBZ-337) to track it and reworded the commit message to refer to that issue. Thanks, @ssiahetiong!

**Pull request closed without merging**